doc/readme.md

```diff
@@ -0,0 +1,131 @@
+# TechTestApp
+
+[![Build Status][circleci-badge]][circleci]
+[![Release][release-badge]][release]
+[![GoReportCard][report-badge]][report]
+[![License][license-badge]][license]
+
+[circleci-badge]: https://circleci.com/gh/vibrato/TechTestApp.svg?style=shield&circle-token=8dfd03c6c2a5dc5555e2f1a84c36e33bc58ad0aa
```

Comment:
Does this token provide any important permissions to the project in CircleCI? In any case, I think it can be revoked as the project is now public

Reply:
Read only token for reading build status :)